pages/register.tsx

```diff
@@ -275,65 +281,174 @@ export default function Register() {
-              {generalQuestions.map((obj, idx) => (
-                <DisplayQuestion key={idx} obj={obj} values={values} onChange={handleChange} />
-              ))}
-
-              <div className="text-2xl py-1 border-b-2 border-black mr-auto mt-8">School Info</div>
-              {schoolQuestions.map((obj, idx) => (
-                <DisplayQuestion key={idx} obj={obj} values={values} onChange={handleChange} />
-              ))}
-
-              <div className="text-2xl py-1 border-b-2 border-black mr-auto mt-8">
-                Hackathon Experience
-              </div>
-              {hackathonExperienceQuestions.map((obj, idx) => (
-                <DisplayQuestion key={idx} obj={obj} values={values} onChange={handleChange} />
-              ))}
-
-              <div className="text-2xl py-1 border-b-2 border-black mr-auto mt-8">Event Info</div>
-              {eventInfoQuestions.map((obj, idx) => (
-                <DisplayQuestion key={idx} obj={obj} values={values} onChange={handleChange} />
-              ))}
-
-              <div className="text-2xl py-1 border-b-2 border-black mr-auto mt-8">Sponsor Info</div>
-              {sponsorInfoQuestions.map((obj, idx) => (
-                <DisplayQuestion key={idx} obj={obj} values={values} onChange={handleChange} />
-              ))}
-
-              {/* Resume Upload */}
-              <label className="mt-4">
-                Upload your resume:
-                <br />
-                <input
-                  onChange={(e) => handleResumeFileChange(e)}
-                  name="resume"
-                  type="file"
-                  formEncType="multipart/form-data"
-                  accept=".pdf, .doc, .docx, image/png, image/jpeg, .txt, .tex, .rtf"
-                />
-                <br />
-              </label>
-
-              {/* Submit */}
-              <div className="my-8">
-                <button
-                  type="submit"
-                  className="mr-auto cursor-pointer px-4 py-2 rounded-md bg-blue-200 hover:bg-blue-300"
-                  onClick={() => setFormValid(!(!isValid || !dirty))}
-                >
-                  Submit
-                </button>
-                {!isValid && !formValid && (
-                  <div className="text-red-600">Error: The form has invalid fields</div>
-                )}
-              </div>
+              {/* General Questions */}
+              {registrationSection == 0 && (
+                <section className="bg-white lg:w-3/5 md:w-3/4 w-full min-h-[35rem] mx-auto rounded-2xl md:py-10 py-6 px-8 mb-8 text-[#4C4950]">
+                  Please fill out the following fields. The application should take approximately 5
+                  minutes.
+                  <h2 className="sm:text-2xl text-xl font-semibold mt-5 sm:mb-3 mb-1">General</h2>
+                  <div className="flex flex-col">
+                    {generalQuestions.map((obj, idx) => (
```

Comment:
borders around the input are lighter in figma.

figma:
<img width="526" alt="Screenshot 2023-03-02 at 1 32 15 PM" src="https://user-images.githubusercontent.com/47117192/222533150-d4f93cc8-6fe3-4022-9eae-37f300fa9db5.png">

urs:
<img width="868" alt="Screenshot 2023-03-02 at 1 33 43 PM" src="https://user-images.githubusercontent.com/47117192/222533397-8c519adf-879d-41e9-a7a8-518ec014cbbd.png">

Reply:
Is the lighter text for placeholder text or is it full actual input? I thought it was just for placeholder but we're not using any placeholder text. I'll change the border tho

Reply:
thats the just the placeholder, i think kelly's design was inconsistent there cause she made me do this for the sign in page:
<img width="593" alt="Screenshot 2023-03-02 at 2 03 22 PM" src="https://user-images.githubusercontent.com/47117192/222539648-c48adcdd-4c33-4011-841f-37b1aabb8e39.png">
<img width="580" alt="Screenshot 2023-03-02 at 2 02 04 PM" src="https://user-images.githubusercontent.com/47117192/222539665-ac990d85-dd93-4f1e-af3b-a4d908056ef9.png">

Reply:
Ignore everything I just commented here I must be dyslexic lol totally misread what you typed